sled-agent/src/bootstrap/early_networking.rs

```diff
@@ -362,15 +363,18 @@ impl<'a> EarlyNetworkSetup<'a> {
         rack_network_config: &RackNetworkConfig,
         switch_zone_underlay_ip: Ipv6Addr,
     ) -> Result<Vec<PortConfig>, EarlyNetworkSetupError> {
+        let log = self.log.new(o!("stage" => SWITCH_ZONE_INIT_STAGE));
```

Comment:
This does feel like the right place for this kind of decision (in contrast to a couple of my comments above) - this is only called when we know we're a scrimlet and are actively initializing the switch zone

Reply:
Fixed, thanks!